snippets/page.json

```diff
@@ -303,10 +303,11 @@
         "page ${1:Id} ${2:PageName}",
         "{",
         "\tPageType = PromptDialog;",
-        "\tCaption = '${3:Draft with Copilot}';",
-        "\tPromptMode = ${4|Prompt,Generate,Content|};",
+        "\tApplicationArea = ${3|All,Basic,Suite,Advanced|};",         
+        "\tCaption = '${4:Draft with Copilot}';",
+        "\tPromptMode = ${5|Prompt,Generate,Content|};",
         "\tExtensible = false;",
-        "\tSourceTable = ${5:TableName};",
+        "\tSourceTable = ${6:TableName};",
```

Comment:
Don't we have to re-index all the tab indexes below too?

Reply:
Yes we should - good catch.